api/src/contracts/cereTokenAbi.json

```diff
@@ -0,0 +1,190 @@
+[
```

Comment:
Where did you take abi?

Reply:
[Etherescan](https://etherscan.io/address/0x2da719db753dfa10a62e140f436e1d67f2ddb0d6#code).